main.js

```diff
@@ -3,7 +3,7 @@
 var visitors = require('./vendor/fbtransform/visitors');
 var transform = require('jstransform').transform;
 var typesSyntax = require('jstransform/visitors/type-syntax');
-var Buffer = require('buffer').Buffer;
+var Buf = require('buffer').Buffer;
```

Comment:
Just add `global Buffer` or whatever the escape is. It's dumb that node puts `Buffer` in the global but such is life. I'd rather the lint escape than write code that looks different from everybody else's.